storagenode/satellites/satellites.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellites
+
+import (
+	"context"
+	"time"
+
+	"storj.io/storj/pkg/storj"
+)
+
+//Status refers to the state of the relationship with a satellites
+type Status = int
+
+const (
+	//Unexpected status should not be used for sanity checking
+	Unexpected Status = iota
+	//Normal status reflects a lack of graceful exit
+	Normal
+	//Exiting reflects an active graceful exit
+	Exiting
+	//ExitedOk reflects a graceful exit that succeeded
+	ExitedOk
+	//ExitedFailed reflects a graceful exit that failed
+	ExitedFailed
+)
+
+// DB works with satellite database
+//
+// architecture: Database
+type DB interface {
```

Comment:
We need a `GetGracefulExit` function or something to allow us to check the status